django/db/templates/db/base.html

```diff
@@ -2,7 +2,21 @@
 
 {% load hms_tags %}
 
+{% block requirejs %}
+    <script>
+    var QUALTRICS_SURVEY_ID = '{{ QUALTRICS_SURVEY_ID }}';
```

Comment:
Part of the move to requirejs modules is eliminating globals. :) I think a better way to get the value into the db/main module would be to have db/main define a `setup` function which takes the survey ID as its only argument, then add an anonoymous function to the `require` call here which calls setup. I'll fork your branch and write it up.